src/admin/content/views/ContentDetail.tsx

```diff
@@ -109,7 +156,7 @@ const ContentDetail: FunctionComponent<ContentDetailProps> = ({ history, match,
 											Omschrijving:
 										</Trans>
 									</BlockHeading>
-									<p>{contentItem.description}</p>
+									<p dangerouslySetInnerHTML={{ __html: contentItem.description }} />
```

Comment:
Indien `contentItem.description` via een WYSIWYG door de gebruiker wordt ingevuld, moeten we hier nog sanitizing voorzien om XSS-vulnerabilities te voorkomen.